Makefile

```diff
@@ -82,15 +91,13 @@ calypso-dev:
 
 # Build desktop bundle
 build-desktop:
-	@NODE_ENV=$(NODE_ENV) NODE_PATH=calypso/server$(ENV_PATH_SEP)calypso/client CALYPSO_SERVER=true npx webpack --config $(THIS_DIR)/webpack.config.js
-
-	@echo "$(CYAN)$(CHECKMARK) Desktop built$(RESET)"
+ifeq ($(NODE_ENV),development)
+	@echo "$(CYAN)$(CHECKMARK) Starting Desktop Server...$(RESET)"
+endif
 
-# Build and watch desktop scripts
-desktop-dev:
-	@echo "$(CYAN)Starting Desktop Server...$(RESET)"
+	@NODE_ENV=$(NODE_ENV) NODE_PATH=calypso$/server$(ENV_PATH_SEP)calypso$/client CALYPSO_SERVER=true npx webpack --config $(THIS_DIR)$(/)webpack.config.js
```

Comment:
any specific reason for `$(/)` here instead of `$/`?

Reply:
This must have slipped through 😇 Thanks!